mava/systems/tf/mad4pg/__init__.py

```diff
@@ -17,7 +17,7 @@
     MAD4PGFeedForwardExecutor,
     MAD4PGRecurrentExecutor,
 )
-from mava.systems.tf.mad4pg.networks import make_default_networks
+from mava.systems.tf.mad4pg.networks import make_1d_conv_networks, make_default_networks
```

Comment:
Do we want this in Mava?

Reply:
See above. I am happy to remove it or leave it.